src/utils/installDependencies.ts

```diff
@@ -0,0 +1,23 @@
+import { spawnSync } from 'child_process'
+
+function installDependencies(projectRoot: string, packageManager: 'npm' | 'yarn' | 'pnpm') {
+  const cmd = (
+    packageManager === 'npm' ?
+      'npm install' :
+      packageManager === 'yarn' ?
+        'yarn' :
+        'pnpm install'
```

Comment:
what is packageManage is null?

Reply:
This function won't except an undefined or null packageManager, this function will only ever get called if there is a selection made to be passed to it.